NER_Data_collection_and_annotation.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<ol><li>are <code>definitely_not_names</code> some common false positives you noticed from spacy?</li><li>I also think line 45 and after would be cleaner if you iterated through both key and value <code>for key, value in conversation.items()</code>. This applies for the other dictionary iterations that leverage key and value</li><li>How effective do you think your logic for identifying street address is (line 52), esp with concerns to false negatives? have you tried any other methods?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meetnaren/ner-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<ol><li>Yes</li><li>You're right, it's embarrassing I didn't think of that in the first place!</li><li>It probably isn't very effective, because I haven't coded up the regex that we talked about for the street addresses yet - that should cast a much wider net.</li></ol>